package.json

```diff
@@ -165,6 +165,7 @@
     "@types/slate": "0.47.11",
     "@types/slate-plain-serializer": "0.7.2",
     "@types/slate-react": "0.22.9",
+    "@types/systemjs": "6.13.1",
```

Comment:
This is the `@types` package rather than systemjs itself. Not sure why they aren't aligned...

```
$ npm view @types/systemjs dist-tags.latest
6.13.1
$ npm view systemjs dist-tags.latest
6.14.1
```